inst/extdata/packages/enmSdmX.yaml

```diff
@@ -0,0 +1,23 @@
+name: "enmSdmX"
+title: "Species Distribution Modeling and Ecological Niche Modeling"
+version: 1.0.1
+author: "Adam B. Smith"
+maintainer: "Adam B. Smith"
+cran: yes
+github: "https://github.com/adamlilith/enmSdmX"
+description: "Implements species distribution modeling and ecological niche modeling, including: bias correction, spatial cross-validation, model evaluation, raster interpolation, biotic "velocity" (speed and direction of movement of a "mass" represented by a raster), interpolating across a time series of rasters, and use of spatially imprecise records. The heart of the package is a set of "training" functions which automatically optimize model complexity based number of available occurrences. These algorithms include MaxEnt, MaxNet, boosted regression trees/gradient boosting machines, generalized additive models, generalized linear models, natural splines, and random forests. To enhance interoperability with other modeling packages, no new classes are created. The package works with 'PROJ6' geodetic objects and coordinate reference systems."
+occ_acquisition: no
+occ_cleaning: no
+env_collinearity: no
+env_interpolation: yes
+bias: yes
+study_region: no
+data_partitioning: no
+calib_general: yes
+calib_ensemble: no
+calib_evaluation: yes
+pred_general: no
+pred_extrapolation: no
+post_processing: yes
```

Comment:
The advantage of the Pull Request from my point of view is the built-in review system that can be done directly on a specific line. For example, here I might ask what post-processing functions are offered by `enmSdmX`?

Reply:
Very nice!  Do you think it would be advisable to ask for a list of relevant functions (or maybe "the 3 most relevant") for each category?  I can see that could get tedious, but it might also ensure people think carefully about each category, too.

Reply:
I think this is a good idea and note that long to read for reviewers. We may improve contributions guidelines accordingly and maybe also write a review guidelines for reviewers.

Reply:
Also as you can see we can define an assignee as an editor and reviewers that already participated to the repository. So I think we have all we need.